packages/patrol_cli/lib/src/coverage/run_code_coverage.dart

```diff
@@ -0,0 +1,190 @@
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
```

Comment:
Don't import `dart:io`, it makes mocking&testing impossible/hard. Prefer `package:file` to `dart:io`. If using some symbols from `dart:io` is  necessary, import `dart:io as io`, to make it clear it's being depended at.

Examples:
- https://github.com/leancodepl/patrol/blob/9f97af3615374ba8d5996e5ba6ad75496d72b7c5/packages/patrol_cli/lib/src/ios/ios_test_backend.dart#L56
- https://github.com/leancodepl/patrol/blob/9f97af3615374ba8d5996e5ba6ad75496d72b7c5/packages/patrol_cli/lib/src/pubspec_reader.dart#L84-L91